src/styles/styles.css

```diff
@@ -502,34 +505,39 @@ section.examples-lists > div {
 
 a.button-link {
   flex-grow: 1;
-  font-size:15px;
+  font-size: 16px;
   color: var(--button-link-color);
   background-color: var(--button-link-background-color);
-  border: 1px solid #302F45;
+  border: none;
   border-radius: 10px;
   font-family: 'Open Sans', sans-serif;
   text-decoration: none !important;
   margin: 10px 5px;
   padding: 24px 24px;
   display: inline-flex;
+  font-weight: 600;
 }
 
 a.button-link:hover {
   background-color: var(--button-link-background-color-hover);
-  border: 1px solid #1664ff;
-  color: white;
+  border: none;
+  color: var(--button-bg-blue);
 }
 
 a.button-link.basic {
-  font-weight: 600;
   align-items: center;
   justify-content: center;
 }
 
 a.button-link.prev {
-  font-weight: 600;
   align-items: center;
   justify-content: center;
+  color: #ffffff;
```

Comment:
I can't see this one very well, especially with the white text. Is there a big reason we need to have back be different than next? I do like the next style change a lot.